tests/stub/driver_execute_query/test_driver_execute_query.py

```diff
@@ -0,0 +1,256 @@
+from nutkit.frontend import (
+    BookmarkManager,
+    Driver,
+    Neo4jBookmarkManagerConfig,
+)
+import nutkit.protocol as types
+from tests.shared import TestkitTestCase
+from tests.stub.shared import StubServer
+
+
+class TestDriverExecuteQuery(TestkitTestCase):
+    required_features = (
+        types.Feature.BOLT_5_0,
+        types.Feature.API_BOOKMARK_MANAGER,
+        types.Feature.API_DRIVER_EXECUTE_QUERY
+    )
+
+    def setUp(self):
+        super().setUp()
+        self._router = StubServer(9000)
+        self._reader = StubServer(9010)
+        self._writer = StubServer(9020)
+        self._driver = None
+
+    def tearDown(self):
+        if self._driver:
+            self._driver.close()
+
+        self._router.reset()
+        self._reader.reset()
+        self._writer.reset()
+        return super().tearDown()
+
+    def test_execute_query_without_params_and_config(self):
+        self._start_server(self._router, "router.script")
+        self._start_server(self._writer, "tx_return_1.script")
+        self._driver = self._new_driver()
+
+        eager_result = self._driver.execute_query("RETURN 1 AS n")
+
+        self.assertEqual(eager_result.keys, ["n"])
+        self.assertEqual(eager_result.records, [
+                         types.Record(values=[types.CypherInt(1)])])
+        self.assertIsNotNone(eager_result.summary)
+
+    def test_execute_query_without_config(self):
+        self._start_server(self._router, "router.script")
+        self._start_server(self._writer, "tx_return_1_with_params.script")
+        self._driver = self._new_driver()
+
+        eager_result = self._driver.execute_query("RETURN 1 AS n", {
```

Comment:
```suggestion
        eager_result = self._driver.execute_query("RETURN $a AS n", {
```

would be more logical (applies to other places too)